force-app/main/default/classes/RD2_OpportunityEvaluation_BATCH.cls

```diff
@@ -201,11 +245,17 @@ public without sharing class RD2_OpportunityEvaluation_BATCH
     */
     @TestVisible
     private String getRDBatchQuery() {
+        String rdFilterWhereClause;
+        if (closedFilterEnabled) {
+            rdFilterWhereClause = 'Status__c != ' + RD2_Constants.STATUS_CLOSED;
```

Comment:
This checks only for the specific "Closed" status, but what if the customer has mapped new statuses to the "Closed" state?